README.md

```diff
@@ -1,89 +1,28 @@
-
-
-## Duties of a Hardware Testing Framework
-OpenHTF attempts to distill the core duties any hardware testing framework must
-perform, handle each one in a clean, sensible fashion, and avoid any additional
-fluff. Those duties are (with example tasks):
-
-  * Manage configuration for each test.
-    * e.g. Load calibration data from a configuration file.
-    * e.g. Use a configuration file to point testers to networked test
-      equipment.
-  * Provide consistent testrun I/O.
-    * e.g. Output records in the same format across all test stations in a
-      manufacturing line, making it easier to write systems to ingest and
-      analyze test results.
-    * e.g. Have a uniform frontend across test stations for intuitive operator
-      interactions.
-    * e.g. Monitor all stations from a central frontend.
-  * Manage test start and execution.
-    * e.g. Plug in a DUT and have a test start automatically.
-    * e.g. Ensure every instance of a test runs the same logic in the same
-      order.
-  * Provide hardware interface tools.
-    * e.g. Provide shared hardware interfaces wrappers for things like USB,
-      UART, GPIO, etc.
-    * e.g. Allow user-written hardware interface wrappers at higher layers of
-      abstraction, like Android ADB.
-
-The package/module structure of OpenHTF is designed to reflect its core duties
-in a clear, obvious fashion.
-
-```
-  .-----------------.
-  |     openhtf     |
-  |-----------------|
-  | Python package. |
-  '-----------------'
-          |
-          |    .---------------------------------.
-          |    |              conf               |
-          |--->|---------------------------------|
-          |    | Reads and stores configuration. |
-          |    '---------------------------------'
-          |
-          |    .-------------------------.
-          |    |           exe           |
-          |--->|-------------------------|
-          |    | Manages test execution. |
-          |    '-------------------------'
-          |
-          |    .----------------------------------------.
-          |    |                   io                   |
-          |--->|----------------------------------------|
-          |    | Manages UI, logging, and test records. |
-          |    '----------------------------------------'
-          |
-          |    .---------------------------------------------------.
-          |    |                       plugs                       |
-          |--->|---------------------------------------------------|
-          |    | Extensions for interfacing with various hardware. |
-          |    '---------------------------------------------------'
-          |
-          |    .-----------------------------------.
-          |    |               util                |
-          '--->|-----------------------------------|
-               | Utility functions and misc tools. |
-               '-----------------------------------'
-```
+## Using OpenHTF
```

Comment:
could use install directions, even if it's just a one-liner referencing `python setup.py install`.  i get this question a lot, even though it's kind of a "just google it" question.